libs/server-sdk/include/launchdarkly/server_side/bindings/c/evaluation_series_context.h

```diff
@@ -0,0 +1,106 @@
+/**
+ * @file evaluation_series_context.h
+ * @brief C bindings for read-only evaluation context passed to hooks.
+ *
+ * EvaluationSeriesContext provides information about a flag evaluation
+ * to hook callbacks. All data is read-only and valid only during the
+ * callback execution.
+ *
+ * LIFETIME:
+ * - All context parameters are temporary - valid only during callback
+ * - Do not store pointers to context data
+ * - Copy any needed data (strings, values) if you need to retain it
+ */
+
+#pragma once
+
+#include <launchdarkly/bindings/c/export.h>
+#include <launchdarkly/server_side/bindings/c/hook_context.h>
+#include <launchdarkly/bindings/c/value.h>
+#include <launchdarkly/bindings/c/context.h>
+
+// No effect in C++, but we want it for C.
+// ReSharper disable once CppUnusedIncludeDirective
+#include <stdbool.h> // NOLINT(*-deprecated-headers)
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct p_LDServerSDKEvaluationSeriesContext* LDServerSDKEvaluationSeriesContext;
```

Comment:
So, there was a mistake with the syntax of the clang-tidy suppression in the base binding implementations. Which result in suppressing all tidy messages.

We are not doing that here, and the result is that these opaque struct definitions were encountering: https://clang.llvm.org/extra/clang-tidy/checks/bugprone/reserved-identifier.html

Which isn't going to be a problem because the compiler isn't going to spontaneously reserve these very specific names.

But to appease it here I have started with `p_`. This first name isn't used by calling code, but we just want it to not look like it should be used either.